src/logic/operate.js

```diff
@@ -0,0 +1,61 @@
+import Big from 'big.js';
+// eslint-disable-next-line consistent-return
+const operate = (numberOne, numberTwo, operation) => {
+  let a;
+  let b;
+
+  if (numberOne) {
+    if (numberOne.includes('.') && numberOne.split('.').length - 1 === 1) {
+      a = new Big(numberOne);
+    } else if (
+      numberOne.includes('.')
+      && numberOne.split('.').length - 1 !== 1
+    ) {
+      return 'Math Error';
+    } else {
+      a = new Big(numberOne);
+    }
+  }
+
+  if (numberTwo) {
+    if (numberTwo.includes('.') && numberTwo.split('.').length - 1 === 1) {
+      b = new Big(numberTwo);
+    } else if (
+      numberTwo.includes('.')
+      && numberTwo.split('.').length - 1 !== 1
+    ) {
+      return 'Math Error';
+    } else {
+      b = new Big(numberTwo);
+    }
+  }
+
+  if (operation === '%') {
+    if (numberOne) {
+      return a.mul(0.01).valueOf();
+    }
+  }
+  if (operation === '-') {
+    return a.minus(b).valueOf();
+  }
+
+  if (operation === '+') {
+    return a.plus(b).valueOf();
+  }
+
+  if (operation === '*') {
+    return a.mul(b).valueOf();
+  }
+
+  if (operation === '÷') {
+    if (numberOne === '0' && numberTwo === '0') {
+      return 'Math Error';
+    }
+    if (numberOne !== '0' && numberTwo === '0') {
+      return 'Math Error';
+    }
+
```

Comment:
There's math error only if `numberTwo` is zero, if `numberOne` is zero, the answer should be zero